privacyidea/api/policy.py

```diff
@@ -97,6 +97,25 @@ def disable_policy_api(name):
     g.audit_object.log({"success": True})
     return send_result(p)
 
+@policy_blueprint.route('/<name>', methods=['PATCH'])
+@log_with(log)
+@prepolicy(check_base_action, request, ACTION.POLICYWRITE)
+def patch_policy_name_api(name):
+    """
+    Rename an existing policy.
+
+    Only the policy’s name is modified; all other attributes remain unchanged.
+
+    :param name: Current name of the policy (from the URL).
+    :jsonparam name: New name to assign to the policy (in the JSON body).
+    :return: Database ID of the renamed policy.
+    """
+    new_name = get_required(request.all_data, "name")
+    check_policy_name(new_name)
```

Comment:
`rename_policy` already does `check_policy_name` so no need to do it here aswell